utils/include/detray/simulation/event_writer.hpp

```diff
@@ -35,7 +35,7 @@ struct event_writer : actor {
                   get_event_filename(event_id, "-measurement-simhit-map.csv")),
               m_meas_smearer(smearer) {}
 
-        uint64_t particle_id = 0u;
+        uint64_t particle_id = -1u;
```

Comment:
This is not safe to do with an unsigned integer type. Is there no way to have the enumeration start at zero?

Reply:
I fixed it